blueprints/addon/index.js

```diff
@@ -86,6 +86,19 @@ module.exports = {
     contents['ember-addon'] = contents['ember-addon'] || {};
     contents['ember-addon'].configPath = 'tests/dummy/config';
 
+    contents.files = [
+      'addon/**/!(.gitkeep)',
+      'addon-test-support',
+      'app/**/!(.gitkeep)',
+      'blueprints',
+      'config/**/!(ember-try.js)',
```

Comment:
In general I think we should move `config/ember-try.js` into `tests/dummy/config/ember-try.js`.

```suggestion
      'config',
```

Reply:
Can ember-try find it there?

Reply:
Not sure, but we should make sure it can (if it can't) 😸